integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -353,4 +353,12 @@ public interface TestConstants {
       + "config/samples/sidb/singleinstancedatabase.yaml";
   public static final String ORACLELINUX_TEST_VERSION =
       getNonEmptySystemProperty("wko.it.oraclelinux.test.version", "7");
+
+  //retry improvement
+  //Defaulting to 120 seconds
+  public static final Long FAILURE_RETRY_INTERVAL_SECONDS =
+      Long.valueOf(getNonEmptySystemProperty("Failure.retry.interval.seconds", "60"));
```

Comment:
Failure.retry.interval.seconds --> failure.retry.interval.seconds to maintain  the property naming convention.

Why are we setting a lower value (60s)  then default (120s)

Reply:
changed.

change from 120s  to 60s is just to lower the value to catch failures easily